proc_DOSY_CPMG.py

```diff
@@ -24,8 +24,7 @@
     fl.next('request 3')
     fl.image(s)
     fl.next('plot indirect 0')
-    fl.plot(s)
-    fl.show();quit()
+    fl.plot(s['indirect',0])
```

Comment:
You had changed this so that what you're plotting differed from the plot -- also the conditional that follows has the same effect as the quit statement.  very bad form to have a quit statement in the middle of code